src/api/Base.js

```diff
@@ -349,8 +354,8 @@ class Base {
                 ...requestData,
             });
             this.successHandler(data);
-        } catch (error) {
-            this.errorHandler(error);
+        } catch (e) {
```

Comment:
Curious for the reason behind this change? I think we usually want to avoid abbreviations.

Reply:
No particular reason. I saw e being used in a few places, so just changed it to be consistent